spikingjelly/activation_based/triton_kernel/flex_sn_inductor/hop.py

```
@@ -135,6 +209,515 @@ def eager_scan(
+
+        step_inputs = tuple(_ensure_contiguous(queue[0]) for queue in step_input_queues)
+        results = core_fn(*step_inputs, *states, *lifted_args)
+        results = tuple(results) if not isinstance(results, torch.Tensor) else (results,)
+        outputs = tuple(_ensure_contiguous(x) for x in results[:num_outputs])
+        next_states = tuple(_ensure_contiguous(x) for x in results[num_outputs:])
+        next_pending_inputs = tuple(
+            _shift_input_queue(queue) for queue in step_input_queues
+        )
+        next_output_acc = tuple(
+            _append_to_tail(buf, out)
+            for buf, out in zip(outputs_acc, outputs)
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)